<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_access</class>
 <widget class="QDialog" name="d_access">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d_access{
    border-image: url(:/bg/resource/bg_access.png);
}

QLabel {
    color: #535151;
    font-family: &quot;黑体&quot;;
}

QLineEdit {
    background-color: white;
    border: 2px solid #868282;
    border-radius: 5px;
}

QLineEdit:hover {
    background-color: #F5EFEF;
    border: 2px solid #868282;
    border-radius: 5px;
}

#pushButton_access_confirm {
    background-color: white;
    border-color: #535151;
    color: #474343;
    border: 2px solid #868282;
    border-radius: 5px;
    border-style: outset;
}

#pushButton_access_confirm:hover{
    background-color: rgb(146,168,209);
    border-color: rgb(146,168,209);
    color: white;
    border: 2px solid rgb(146,168,209);
    border-radius: 5px;
    border-style: outset;
}
 
#pushButton_access_return {
    background-color: white;
    border-color: #535151;
    color: #474343;
    border: 2px solid #868282;
    border-radius: 5px;
    border-style: outset;
}

#pushButton_access_return:hover{
    background-color: rgb(146,168,209);
    border-color: rgb(146,168,209);
    color: white;
    border: 2px solid rgb(146,168,209);
    border-radius: 5px;
    border-style: outset;
}

</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_access_Name">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>400</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_access_Key1">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>460</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_access_Key2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>520</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_access_confirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_access_return">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_access_info">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>580</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_access_text1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_access_text2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密  码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_access_text3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确  认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
